pkg/api/api.go

```diff
@@ -201,8 +200,15 @@ func (a *API) RegisterDistributor(d *distributor.Distributor, pushConfig distrib
 	a.RegisterRoute("/ha-tracker", d.HATracker, false, "GET")
 }
 
+type ingester interface {
```

Comment:
Shouldn't be exported?

Reply:
For what I'm trying to do it's not necessary to export it. My goal is just to pass `RegisterIngesters()` an alternative struct which satisfies the same interface as the "original" ingester. Or is there another reason to export it?

Reply:
I don't think export is necessary, but a comment explaining why we do this would be nice. Otherwise it looks like magic.

Reply:
ok, good point, I'll add one